xbmc/music/windows/GUIWindowMusicNav.h

```diff
@@ -38,6 +38,7 @@ class CGUIWindowMusicNav : public CGUIWindowMusicBase
   std::string GetStartFolder(const std::string &url) override;
 
   bool GetSongsFromPlayList(const std::string& strPlayList, CFileItemList &items);
+  std::string GetQuickpathName(const std::string& strPath) const;
```

Comment:
Rebase error - remove

Reply:
OK, I'll fix all those.  Probably I should have realised as I do tend to check your commits and associated comments every so often.  Looking forward to the day I get this git (and that is the proper word for it !!) stuff right, if you can bear with me that long :)